### PR TITLE
Luthier tools and woodworked instrument recipes

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Content "Crafting Musical Instruments"

#### Purpose of change
Make musical instruments craftable with the correct tools and proficiencies.

#### Describe the solution
Start with adding a luthier quality and toolset using it.  Then start on recipes.   

#### Describe alternatives you've considered
None

#### Testing
Loaded it up and tested the hobby, sad I can't add specific recipes to the luthier hobby, crafted a krar, got interrupted by my opiate addiction, realized that needs more snippets, no don't scope creep this PR.  Tested and works as expected.

### CODE REVIEW

File: data/json/recipes/recipe_music_instruments.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "byproducts": [ [ "splinter", 89 ] ],
+    "components": [
+      [ [ "tonewood", 2 ] ],
+      [ [ "superglue", 2 ] ],
+      [ [ "wood_panel", 1 ] ],
+      [ [ "piano_wire", 6 ] ],
+      [ [ "tuning_pegs", 6 ] ]
+    ]
+  }
+]
```
Comment (Maleclypse) [inline data/json/recipes/recipe_music_instruments.json:65]:
```suggestion
]
```

File: data/json/items/resources/wood.json [hunk 2]
```diff
@@ -132,6 +132,41 @@
     "flags": [ "FRAGILE_MELEE", "FIREWOOD" ],
     "melee_damage": { "bash": 16 }
   },
+  {
+    "type": "GENERIC",
+    "id": "tonewood",
+    "name": { "str": "tonewood" },
```
Comment (Maleclypse) [inline data/json/items/resources/wood.json:138]:
```suggestion
    "name": { "str": "tonewood", "str_pl": "blocks of tonewood" },
```

File: data/json/items/resources/wood.json [hunk 3]
```diff
@@ -132,6 +132,41 @@
...
+    "to_hit": { "grip": "none", "length": "long", "surface": "any", "balance": "uneven" },
+    "price": 5000,
+    "price_postapoc": 100,
+    "flags": [ "FIREWOOD" ],
+    "melee_damage": { "bash": 6 }
+  },
+  {
+    "type": "GENERIC",
+    "id": "tuning_pegs",
+    "name": { "str": "tuning pegs" },
```
Comment (Maleclypse) [inline data/json/items/resources/wood.json:156]:
```suggestion
    "name": { "str": "tuning peg", "str_pl": "tuning pegs" },
```

File: data/json/items/resources/wood.json [hunk 4]
```diff
@@ -132,6 +132,41 @@
...
+    "to_hit": { "grip": "none", "length": "long", "surface": "any", "balance": "uneven" },
+    "price": 5000,
+    "price_postapoc": 100,
+    "flags": [ "FIREWOOD" ],
+    "melee_damage": { "bash": 6 }
+  },
+  {
+    "type": "GENERIC",
+    "id": "tuning_pegs",
+    "name": { "str": "tuning peg", "str_pl": "tuning pegs" },
```
Comment (Maleclypse) [inline data/json/items/resources/wood.json:156]:
```suggestion
    "name": { "str": "tuning peg" },
```

File: data/json/recipes/recipes.json [hunk 5]
```diff
@@ -116,6 +116,11 @@
       "CSC_ANIMALS_EQUINE STORAGE"
     ]
   },
+  {
+    "type": "recipe_category",
+    "id": "CC_MUSIC",
+    "recipe_subcategories": [ "CSC_ALL", "CSC_MUSIC_STRING", "CSC_MUSIC_WIND", "CSC_MUSIC_PERCUSSION", "CSC_MUSIC_KEYBOARD" ]
```
Comment (Maleclypse) [inline data/json/recipes/recipes.json:122]:
```suggestion
    "recipe_subcategories": [ "CSC_ALL", "CSC_MUSIC_STRING", "CSC_MUSIC_WIND", "CSC_MUSIC_PERCUSSION" ]
```

File: data/json/items/tool/musical_instruments.json [hunk 6]
```diff
@@ -317,6 +317,70 @@
...
+    "use_action": [ { "type": "play_instrument" } ],
+    "tick_action": {
+      "type": "musical_instrument",
+      "volume": 28,
+      "fun": -3,
+      "fun_bonus": 2,
+      "speed_penalty": 18,
+      "description_frequency": 20,
+      "player_descriptions": [
+        "You play a beat on your drum. Pa rum pa pum pum.",
```
Comment (Maleclypse) [inline data/json/items/tool/musical_instruments.json:345]:
```suggestion
        "You play a beat on your drum.  Pa rum pa pum pum.",
```

File: data/json/items/tool/musical_instruments.json [hunk 7]
```diff
@@ -317,6 +317,70 @@
...
```
Comment (Maleclypse) [inline data/json/items/tool/musical_instruments.json:377]:
```suggestion
        "You play a beat on your drum.  Pa rum pa pum pum.",
```

**Comment by Fris0uman** [comment]
Maybe the proficiency should be added to a hobby? Otherwise it's going to be pretty hard to get it from zero after chargen

**Comment by I-am-Erk** [comment]
I don't think it belongs in this PR but there's probably some more proficiencies to add around instrument crafting at some point.

**Comment by Maleclypse** [comment]
> Maybe the proficiency should be added to a hobby? Otherwise it's going to be pretty hard to get it from zero after chargen

I'll do that and try to get this back on target with some of my other languishing PRs.

File: data/json/items/tool/musical_instruments.json [hunk 8]
```diff
@@ -317,6 +317,81 @@
     },
     "melee_damage": { "bash": 9 }
   },
+  {
+    "id": "krar",
+    "looks_like": "krar_golden",
+    "type": "TOOL",
+    "category": "tools",
```
Comment (Maleclypse) [inline data/json/items/tool/musical_instruments.json:324]:
```suggestion
```

File: data/json/items/tool/musical_instruments.json [hunk 9]
```diff
@@ -297,8 +297,8 @@
     "name": { "str": "golden krar" },
     "description": "A shiny golden krar, similar to a harp.",
     "weight": "13000 g",
-    "volume": "2500 ml",
-    "longest_side": "60 cm",
+    "volume": "1130973 ml",
+    "longest_side": "1 m",
```
Comment (Maleclypse) [inline data/json/items/tool/musical_instruments.json:301]:
```suggestion
    "longest_side": "100 cm",
```

**Pull request merged**